packages/lb-annotation/src/core/pointCloud/store/index.ts

```diff
@@ -273,13 +295,37 @@ class PointCloudStore {
   // Save temporary data to pointCloud Store.
   public addStash2Store() {
     if (this.cacheSegData) {
+      if (
+        this.cacheSegData.coverPoints &&
+        this.cacheSegData.coverPoints.length !== 0 &&
+        this.segmentCoverMode === EPointCloudSegmentCoverMode.Cover
+      ) {
+        this.updateCoverPoints(this.cacheSegData.coverPoints);
+      }
       this.segmentData.set(this.cacheSegData.id, this.cacheSegData);
 
       this.cacheSegData = undefined;
       this.syncCacheMsg();
     }
   }
 
+  public updateCoverPoints = (coverPoints: Float32Array = new Float32Array([])) => {
+    this.segmentData.forEach((seg, id) => {
+      if (id !== this.cacheSegData?.id) {
+        const newPoints = this.splitPointsFromPoints(seg.points, coverPoints);
+        seg.points = newPoints;
+
+        // update points in threeJs
+        const originPoints = this.scene.getObjectByName(id) as THREE.Points;
+        if (originPoints) {
+          originPoints.geometry.setAttribute('position', new THREE.Float32BufferAttribute(newPoints, 3));
+          originPoints.geometry.attributes.position.needsUpdate = true;
+        }
+      }
+    });
+    this.renderer.render(this.scene, this.camera);
```

Comment:
Store doesn't directly execute redner. You can use `this.emit('reRender3d');`

Reply:
done